addons/website/static/src/js/menu/content.js

```diff
@@ -339,6 +345,26 @@ var PagePropertiesDialog = weWidgets.Dialog.extend({
     // Handlers
     //--------------------------------------------------------------------------
 
+    /**
+     * @private
+     */
+    _onMetaDescriptionChanged: function (ev) {
+        var $textarea = $(ev.target);
+        var $warning = $textarea.next('div#website_meta_description_warning');
+        var length = $textarea.val().length;
+        if (length > this.maxRecommendedDescriptionSize || length < this.minRecommendedDescriptionSize) {
```

Comment:
As 0 length description is acceptable, can you add check condition if not length ?
The preview is corretly pointing the fact an empty description will be handled and set automatically

Reply:
Ok